commons/zenoh-shm/src/lib.rs

```diff
@@ -228,3 +228,20 @@ impl ZSliceBuffer for ShmBufInner {
         self
     }
 }
+
+#[derive(Debug)]
+pub struct ShmBufInfoBuffer(pub Vec<u8>);
```

Comment:
I believe `ShmBufInfoBuffer` can simply be a wrapper around a `ZSlice`?

Reply:
You cannot use a wrapper around ZSlice, otherwise you would not be able to use it in `ZBuf`.